test/browser_tests/spec_suites/careers-application-process.js

```diff
@@ -52,4 +52,31 @@ describe( 'The Application Process Page', function() {
     expect( page.relatedLinksSection.isPresent() ).toBe( true );
   } );
 
+  describe( '(Video Player)', function() {
+    it( 'should be present', function() {
+      expect( page.videoPlayer.isPresent() ).toBe( true );
+    } );
+
+    it( 'display show an image when the page loads', function() {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2103/commits/6528180aee0b99e1ad2bf2e57c032709f1f35ea3.